charts/gateway/templates/gemfire/embedded/gemfire-locator-statefulset.yaml

```diff
@@ -49,10 +49,39 @@ spec:
       annotations: {{- toYaml .Values.podAnnotations | nindent 8 }}
     {{- end }}
     spec:
+      {{- if .Values.affinity }}
+      affinity: {{- toYaml .Values.affinity | nindent 12 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- toYaml .Values.tolerations | nindent 12 }}
+      {{- end }}
+      {{- if .Values.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- toYaml .Values.topologySpreadConstraints | nindent 12 }}
+      {{- end }}
+      {{- if .Values.podSecurityContext }}
+      securityContext: {{- toYaml .Values.podSecurityContext | nindent 12 }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 12 }}
+      {{- end }}
+      {{- if .Values.global.schedulerName }}
+      schedulerName: {{ .Values.global.schedulerName }}
+      {{- end }}
+      {{- if .Values.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{.Values.terminationGracePeriodSeconds }}
+      {{- end }}
       containers:
         - name: {{ .Chart.Name }}-gemfire-locator
           image: {{ .Values.config.gemfire.embedded.externalLocators.image.registry }}/{{ .Values.config.gemfire.embedded.externalLocators.image.repository }}:{{ .Values.config.gemfire.embedded.externalLocators.image.tag }}
           imagePullPolicy: {{ .Values.config.gemfire.embedded.externalLocators.image.pullPolicy }}
+          {{- if .Values.containerSecurityContext }}
+          securityContext: {{- toYaml .Values.containerSecurityContext | nindent 12 }}
+          {{- end }}
+          {{- if .Values.lifecycleHooks }}
+          lifecycle: {{- toYaml .Values.lifecycleHooks | nindent 12 }}
+          {{- end }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
```

Comment:
the global resources values is for the gateway container. if gemfire resource is generally expected to be equal to gateway container, then they care share the same values, but since gemfire appears to be very different, probably should have separate values, like pmtagger is very low resource
e.g.
https://github.com/CAAPIM/apim-charts/blob/develop/gateway/charts/gateway/values.yaml#L89
gemfire is opposite, appear to require much more than gateway.